src/inventory/interfaces_test.go

```diff
@@ -195,3 +195,9 @@ var _ = Describe("Interfaces", func() {
 		}
 	})
 })
+
+func newInterface(mtu int, name string, macAddr string, flags net.Flags, addrs []string, speedMbps int64, interfaceType string) *util.MockInterface {
```

Comment:
``newInterface`` seems like the real deal, maybe:
```suggestion
func newMockInterface(mtu int, name string, macAddr string, flags net.Flags, addrs []string, speedMbps int64, interfaceType string) *util.MockInterface {
```